mqtt3/1.0/modules/ROOT/edits/mqtt3-connector-lwt-message.adoc

```diff
@@ -0,0 +1,49 @@
+= Configure the Last Will And Testament Messages for MQTT Connector
+
+In this example, you set a last will and testament (LWT) message for the Anypoint Connector for MQTT (MQTT Connector) to ensure that if the client disconnects temporarily or permanently from the broker, a message is sent to a specific LWT topic, notifying all its subscribers of the client’s disconnection. The LWT message is set at the time the connection to the broker is established.
+
+Configure the LWT message in the configuration element and specify whether or not you want the broker to retain that message.
+If an MQTT message is set for retention, the broker stores the last message marked as retained that arrived at a specific topic. If no listeners are subscribed to a matching topic at the time the message is sent, the broker stores the message until a listener connects to that topic. Then, the broker delivers the listener to the last message retained for that topic.
```

Comment:
This text confuses me.
The previous sentence passively states that the LWT **is set** during connection with a broker.
Then the first sentence here tells us that **we configure** the LWT and whether the broker retains it (why would it not, if we bother to configure it?)
The next sentence refers to **an MQTT message** set for retention: is this the LWT or some other message? And is this MQTT message the last message that arrived at the topic?
The following and final two sentences jump to the listener part of the retention; it might be clearer to stick with the topic of retention and clarify: if there are listeners, do they automatically get the retained MQTT/LWT message? And otherwise, the MQTT/LWT message is delivered when a listener subscribes?
I think I would understand this better if I saw an illustration of the process, with numbered steps.

Reply:
Also, as always, if this is just my confusion due to being unfamiliar with the UI, then feel free to dismiss these questions (after evaluating, of course).
But I wonder...this is a task topic, and this conceptual info might belong elsewhere, in a topic about what an LWT is and how it works, rather than in a task about configuration steps.